### PR TITLE
Fix error in file references.

### PR DESCRIPTION
**Pull request by prettyxw**

### CODE REVIEW

File: expander.go [hunk 1]
```diff
@@ -231,6 +245,37 @@ func nextRef(startingNode interface{}, startingRef *Ref, ptr *jsonpointer.Pointe
 	return ret
 }
 
+func normalizeFileRef(ref *Ref, relativeBase string) *Ref {
+	refURL := ref.GetURL()
+
+	if !strings.HasPrefix(refURL.String(), "#") {
```
Comment (casualjim) [inline expander.go]: perhaps it's a bit cleaner if you return when it has the prefix, reduces 1 level of nesting

Comment (prettyxw) [inline expander.go]: Yeah, good idea.

**Comment by prettyxw** [comment]
Updated.

**Comment by casualjim** [comment]
thanks :100:

**Comment by prettyxw** [comment]
You're welcome. 🌚

**Pull request merged**